cli/file/definition.go

```diff
@@ -0,0 +1,38 @@
+package file
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/definition"
+	"gopkg.in/yaml.v2"
+)
+
+func LoadDefinition(file string) (definition.Test, error) {
+	fileBytes, err := os.ReadFile(file)
+	if err != nil {
+		return definition.Test{}, fmt.Errorf("could not read file %s: %w", file, err)
```

Comment:
nit: a more specific message could help debugging, like `could not read test definition file...`